buckw

```diff
@@ -213,8 +213,8 @@ setupBuckRun ( ) {
 
 # Handle parameters and flags
 handleParams ( ) {
-    # Go directly to kill. Do not run okbuck.
-    if [[ "kill" == $1 ]]; then
+    # Go directly to kill or --help. Do not run okbuck.
+    if [[ "kill" == $1 || "--help" == $2 ]]; then
```

Comment:
I think `--help` need not be the second argument always. Maybe it is best to check if the `$@` matches `--help` instead of doing it on a positional argument?